up.sh

```diff
@@ -39,5 +39,36 @@ sed -i "s/replace_with_ip/$NFS_SRV_IP/g"  ./netboot/menu.ipxe
 wget https://cloud.3mdeb.com/index.php/s/UQQVYrNIhg7ddwj/download -O kernels.tar.gz
 tar -xzvf kernels.tar.gz -C ./netboot && rm kernels.tar.gz
 
+# download root file systems
+wget https://cloud.3mdeb.com/index.php/s/9b8h6WmJcNsuB57/download -O debian-stable.tar.gz
+wget https://cloud.3mdeb.com/index.php/s/fzQ2FaRTdMvzXqO/download -O xen.tar.gz
+wget https://cloud.3mdeb.com/index.php/s/AQuUdsYkBzO9UJz/download -O core.tar.gz
+wget https://cloud.3mdeb.com/index.php/s/rUZPwRHOjxpSxN4/download -O voyage-0.11.0_amd64.tar.gz
+
+NFS_EXPORT_DIR="./nfs-export"
+mkdir -p $NFS_EXPORT_DIR
+
+# extract debian
+DEBIAN_DIR="$NFS_EXPORT_DIR/debian"
+tar -xvpzf debian-stable.tar.gz -C $NFS_EXPORT_DIR --numeric-owner
+mv $NFS_EXPORT_DIR/debian-stable $DEBIAN_DIR
+
+# extract xen
+XEN_DIR="$NFS_DIR/xen"
+tar -xvpzf xen.tar.gz -C $NFS_EXPORT_DIR --numeric-owner
+
+# extract voyage
+VOYAGE_DIR="$NFS_EXPORT_DIR/voyage"
+tar -xzvf voyage-0.11.0_amd64.tar.gz -C $VOYAGE_DIR
+
+# extract core ??
+tar -xvpzf core.tar.gz -C ./netboot
+
+# remove
+rm voyage-0.11.0_amd64.tar.gz
+rm debian-stable.tar.gz
+rm xen.tar.gz
+rm core.tar.gz
+
 # start containers
 docker-compose up
```

Comment:
We have to somehow resolve conflict with port bindings on host. I got:

```
ERROR: for nfs-server  Cannot start service nfs-server: b'driver failed programming external connectivity on endpoint nfs-server (e2d2855b08c73af0e79b0d6854291cc15255d726b21059a8a647daf8c817903f): Error starting userland proxy: listen tcp 0.0.0.0:111: bind: address already in use'
```

How I resolved it earlier: https://github.com/3mdeb/pxe-server/blob/master/start.sh#L11

Reply:
Sure, what I am suggesting is to add information, that such error may occur and advise a possible fix. Since this is a public repository, the PXE server may be used by anyone.

Reply:
done